src/pages/Login.jsx

```diff
@@ -1,36 +1,70 @@
 import React from 'react';
 import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
+import { Redirect } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
-function Login() {
-  return (
-    <Form>
-      <Form.Group controlId="formBasicEmail">
-        <Form.Label>Email address</Form.Label>
-        <Form.Control
-          data-testid="email-input"
-          type="email"
-          placeholder="Enter email"
-        />
-      </Form.Group>
-      <Form.Group controlId="formBasicPassword">
-        <Form.Label>Password</Form.Label>
-        <Form.Control
-          data-testid="password-input"
-          type="password"
-          placeholder="Password"
-        />
-      </Form.Group>
-      <Button
-        data-testid="login-submit-btn"
-        variant="primary"
-        type="submit"
-      >
-        Entrar
-      </Button>
-    </Form>
-  );
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      isValidLogin: false,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    const { email } = this.state;
+    const pessoa = { email };
+    localStorage.setItem('mealsToken', 1);
+    localStorage.setItem('cocktailsToken', 1);
+    localStorage.setItem('user', JSON.stringify(pessoa));
+    this.setState({ isValidLogin: true });
+  }
+
+  render() {
+    const { email, password, isValidLogin } = this.state;
+    const quanty = 6;
```

Comment:
Fala igson, acho que poderiamos colocar um nome mais descritivo para esta constante. O que acha?
```suggestion
    const minPasswordLength = 6;
```